lib/grunt/task.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+// Keep track of the number of log.error() calls.
+var errorcount,
+
+  // The last specified tasks message.
+  lastInfo;
```

Comment:
With 
```js
"one-var": [
"one-var": [
  "error",
  {
    "uninitialized": "always",
    "initialized": "never"
  }
]
```

It doesn't allow initialized values with different `var` keywords. I believe it satisfy the idea in the blog post you mentioned (see "In summary" at the end) - 

> My rule is as-follows: I’ll put multiple comma-separated declarations in a single var statement, but only if they don’t have assignments. 

```js
// SIMPLE AND SEXY

var foo, bar, baz;
var abc = 1;
var def = 2;
```

In any case, if you want me to change this rule or omit it, I can do that too